GameLogic/player_management/Equipment.cs

```diff
@@ -75,16 +88,26 @@ public class Weapon: Item
         WeaponType = weaponType;
         Attack = attack;
     }
-    public WeaponType WeaponType { set; get; }
-    public int Attack { set; get; }
+    public WeaponType WeaponType { get; set; }
+    public int Attack { get; set; }
 }
 
 public class Equipment
 {
-    public Armour EquippedArmour(Chestplate chest = null, Leggins legs = null, Helmet helmet = null)
+    public Equipment()
     {
-        return new Armour(chest, legs, helmet);
+        EquipArmour();
```

Comment:
?

Reply:
after initializing a player we initialize its armour but without any elements

Reply:
Should it be in the constructor of this class? Is it better to make the method static?